sdcm/cluster.py

```diff
@@ -3878,9 +3898,13 @@ def get_monitoring_data_stack(self, node):
                                                           archive_name,
                                                           os.path.basename(self.monitor_install_path)),
                          ignore_status=True)
+        if os.path.exists(archive_full_path):
```

Comment:
how can we get to the state that we how more than one Prometheus data file?

Reply:
when reuse cluster
when for some reasons the collect logs failed, and run collect logs again

Reply:
We have unique test dir for the test run. Look on the code in `sdcm.cluster.Setup#logdir`
1. Reuse cluster doesn't store logs in the same dir
2. When running collect logs, again and again, it also shouldn't store in the same dir
So I still don't understand how it can happen
(any way we shouldn't delete collected test data from previous runs)

Reply:
@bentsi , 
Went through the code again. 
we get the state when we have 2 archive files of Prometheus data when method update_test_details called first time(or any times in tests) and then when we collect logs in teardown. Because the archive files have the same name, latest archive could n't be created.
We remove only the archive file, which is contain prometheus data, data itself are saved.
Any next snapshot of prometheus data or creating archive, will contain all previos data. so we don't lose anything

Reply:
> @bentsi ,
> Went through the code again.
> we get the state when we have 2 archive files of Prometheus data when method update_test_details called first time(or any times in tests) and then when we collect logs in teardown. Because the archive files have the same name, latest archive could n't be created.
> We remove only the archive file, which is contain prometheus data, data itself are saved.
> Any next snapshot of prometheus data or creating archive, will contain all previos data. so we don't lose anything